javaslang/src/test/java/javaslang/collection/IntMap.java

```diff
@@ -23,7 +23,7 @@
     @SuppressWarnings("unchecked")
     public static <T> IntMap<T> of(Map<Integer, T> original) {
         return original.isEmpty() ? (IntMap<T>) EMPTY
-                                  : new IntMap<>(original);
+                : new IntMap<>(original);
```

Comment:
Than please format only the `VCS changed` lines :) (or manually revert the ones that were better formatted before)